client/dive-common/components/RunPipelineMenu.vue

```diff
@@ -49,10 +50,27 @@ export default defineComponent({
     const successMessage = computed(() => (
       `Started ${selectedPipe.value?.name} on ${props.selectedDatasetIds.length} dataset(s).`));
 
+    const checkCamNumber = async () => {
+      if (props.subTypeList.length === props.subTypeList.filter((item) => item === null).length) {
+        return 1;
+      } if (props.selectedDatasetIds.length === 1) {
+        const meta = await loadMetadata(props.selectedDatasetIds[0]);
+        if (meta.multiCamMedia) {
+          return Object.keys(meta.multiCamMedia.cameras).length;
+        }
+      }
+      return 1;
+    };
     onBeforeMount(async () => {
+      camNumber.value = await checkCamNumber();
       unsortedPipelines.value = await getPipelineList();
     });
 
+    // On initial load from import subTypeList changes
+    watch(() => props.subTypeList, async () => {
+      camNumber.value = await checkCamNumber();
```

Comment:
Probably want to avoid sending off all these API requests.  Is there another way this could be done?